pallets/block-reward/src/types.rs

```diff
@@ -85,34 +91,42 @@ pub struct RewardDistributionConfig {
 	/// Base percentage of reward that goes to treasury
 	#[codec(compact)]
 	pub treasury_percent: Perbill,
-	/// Percentage of rewards that goes to dApps
+	/// Percentage of rewards that goes to DePIN apps
 	#[codec(compact)]
-	pub dapps_percent: Perbill,
-	/// Percentage of reward that goes to collators
+	pub depin_percent: Perbill,
+	/// Percentage of reward that goes to collators and delegators
 	#[codec(compact)]
-	pub collators_percent: Perbill,
-	/// Percentage of reward that goes to lp users
+	pub collators_delegators_percent: Perbill,
+	/// Percentage of reward that goes to coretime
 	#[codec(compact)]
-	pub lp_percent: Perbill,
-	/// Percentage of reward that goes to machines
+	pub coretime_percent: Perbill,
+	/// Percentage of reward that goes to subsidization pool
 	#[codec(compact)]
-	pub machines_percent: Perbill,
+	pub subsidization_pool_percent: Perbill,
 	/// Percentage of reward that goes to parachain lease fund
 	#[codec(compact)]
 	pub parachain_lease_fund_percent: Perbill,
+	/// Percentage of rewards that goes to DePIN staking
+	#[codec(compact)]
+	pub depin_staking_percent: Perbill,
+	/// Percentage of rewards that goes to DePIN incentivization
+	#[codec(compact)]
+	pub depin_incentivization_percent: Perbill,
 }
 
 impl Default for RewardDistributionConfig {
 	/// `default` values based on configuration at the time of writing this code.
 	/// Should be overriden by desired params.
 	fn default() -> Self {
 		RewardDistributionConfig {
-			treasury_percent: Perbill::from_percent(15),
-			dapps_percent: Perbill::from_percent(45),
-			collators_percent: Perbill::from_percent(10),
-			lp_percent: Perbill::from_percent(20),
-			machines_percent: Perbill::from_percent(5),
-			parachain_lease_fund_percent: Perbill::from_percent(5),
+			treasury_percent: Perbill::from_percent(25),
+			depin_percent: Perbill::from_percent(0),
```

Comment:
Why the default DEPIN percentage is 0? Or should we remove it? In your spec, I saw the depin_precentage is composed by the depin_staking_precetange and depin_incentivization_precentage

Reply:
yes i can remove it. I left it here in the first place in case we decide to put staking and incentivization in the same part in the future.

Reply:
Let us remove it because if we leave that, it'll make people feel confused. To me, there are two different solutions.
1. Keep the depin percentage and move the depin_staking_precentage and depin_incentivization_percentage to the future depin-related pallet
2. Keep the depin_staking_percentage and depin_incentivization_perenctage in the block reward configuration

According to our discussion yesterday, let us follow the second solution, and please remember to update your spec